internal/pkg/print/debug.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stackitcloud/stackit-sdk-go/core/config"
 )
 
-var defaultHTTPHeaders = []string{"Accept", "Content-Type", "Content-Length", "User-Agent", "Date", "Referrer-Policy"}
+var defaultHTTPHeaders = []string{"Accept", "Content-Type", "Content-Length", "User-Agent", "Date", "Referrer-Policy", "Traceparent"}
```

Comment:
this is not case sensitive right? The header is just `traceparent`

Reply:
It is but I tested and the header is `Traceparent` with a capital T.
![image](https://github.com/stackitcloud/stackit-cli/assets/33052643/6f2d092e-b36e-4308-a688-6fad281b2e03)

Reply:
Without capitalization nothing appears

Reply:
I was checking in Postman, so I guess Postman is making it everything lower-case

Reply:
it's fine like this then for the scope of this PR